### DISCUSSION TITLE
Sometimes I get no honor for solving a kata

### DISCUSSION THREAD

**Discussion by Yang-Wei-Ting** [codewars/codewars.com#2761] [Q&A] [answered]
I believe that I never forfeit a single kata.
The most recent kata that I completed but received no honor is [Snail](https://www.codewars.com/kata/reviews/54aebf4d1ea50a119c0001a9/groups/62ea658d2eae0100012e21c2).

**Comment by Blind4Basics** [comment, marked as answer]
Hi,

This kata shows up as forfeited in your profile. Note that entering the edit panel is the same as forfeiting a kata. Might be you entered it by mistake. You opened the trainer of this kata 600+ days ago, so maybe you just don't remember?

**Reply by Yang-Wei-Ting** [reply]
> Note that entering the edit panel is the same as forfeiting a kata.

OMG it must be the reason :(

Is it possible for me to know the full list of these non-intentionally forfeited katas, so that I won't expect honor gain from them in the future?

**Reply by Blind4Basics** [reply]
Unfortunately, it's very hard to get that kind of info. As for now, only admins can gather the data (requires a specific DB request)

**Reply by EloiseRosen** [reply]
One quick way to check for a specific kata is seeing if you can access the solutions page, or whether you get the "Solutions have been withheld" page instead. If you can see the solutions (and there's no "completed" checkmark after the kata title at the top), that suggests you've forfeited that kata.

**Reply by Blind4Basics** [reply]
@EloiseRosen : that can help, but that's unfortunately not always reliable. Some actions forfeit in any language, some in one language only, in some situation future translations are automatically forfeited, but old aren't, ...

So if no one goes into details, yeah, when you can see the solution of a language you didn't complete, there is around 95% of chances that you won't get any progress in any language anymore.

**Reply by Yang-Wei-Ting** [reply]
It would be nice to have a popup window like
```
Are you sure that you want to edit this kata?
Doing so without completing it first means forfeiting the kata.
```
or something like that.
